ui/tests/add-btc.spec.js

```diff
@@ -48,12 +48,12 @@ describe('User Settings', function () {
     await this.app.client.click('.jconfirm-buttons>button')
 
     // wait for popup modal, then close it
-    await this.app.client.waitForExist('.jconfirm-box.jconfirm-type-green.jconfirm-type-animated', 5000)
+    await this.app.client.waitForExist('.jconfirm-box', 5000)
     await this.app.client.execute(function () {
         $('.jconfirm').remove()
     })
     // wait for the other modal popup, then close it
-    await this.app.client.waitForExist('.jconfirm-box.jconfirm-hilight-shake.jconfirm-type-animated.jconfirm-type-green', 5000)
```

Comment:
I may be wrong, but I put the full class path for a reason. The order with which these popups appear is not guaranteed and I needed a unique way to address each one. So I think the full class path should be there.

Also @mykeels do you know of any way to abstract this functionality which we use in every tests somehow? Would normal functions work here?